climatedata_api/download.py

```diff
@@ -113,7 +118,15 @@ def output_netcdf(ds, encoding, format):
     return f
 
 
-def check_points_or_bbox(points, bbox, month=None):
+def check_points_or_bbox(points: list[list[float, float]], bbox: list[float, float, float, float], month: str=None):
```

Comment:
Avec le typing `list`, on ne peut pas spécifier le type des éléments individuels. Pour le faire, il faut utiliser `Tuple` (de `from typing`).
```suggestion
def check_points_or_bbox(
    points: list[Tuple[float, float]], bbox: Tuple[float, float, float, float],
    month: str=None
):
```